Dockerfile

```diff
@@ -1,5 +1,11 @@
-FROM python:2
+FROM python:3
 
-RUN pip install ansible boto boto3 netaddr
+LABEL maintainer="EMnify <devops@emnify.com>" \
```

Comment:
Use an existing address or create that one.

Reply:
Can you recommend a central mail address to use for such cases?

Reply:
erm.. I'm not sure, why one needs email addresses here :-) The world meets on Github :D

In doubt I would use `no-reply@emnify.com` (to make clear, it doesn't exist) or something like `opensource@emnify.com`.. again, just the question who wants to read that..

I have no clear opinion on that..

Reply:
well then I suggest to use `EMnify (https://github.com/EMnify)` - let me try if it works :-)